libraries/adaptive-expressions/tests/expressionParser.test.js

```diff
@@ -887,22 +871,40 @@ describe('expression parser functional test', () => {
     // `new Date()` done by both the test and by ExpressionEvaluator return the exact same time.
     it('should appropriately evaluate time-related expressions', () => {
         // Freeze the system clock.
-        // The expected date in MM-DD-YY HH is 01-01-20 00
+        // The expected date in MM-DD-YY HH is 08-01-20 00
```

Comment:
Let's make the comment a little more clear since the tests now include mm:ss:

```suggestion
        // The expected date in the format, MM-dd-yy HH:mm:ss, is 08-01-20 00:12:20
```

Turns out I also had some of the [upper/lowercasing](https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html) wrong in my original comment.